### DISCUSSION TITLE
Semantic Slots and Theme Schemas

### DISCUSSION THREAD

**Discussion by Wezea** [microsoft/fluentui#20625] [Q&A] [answered]
Hi everyone 👋 

Are there any plans to finalize the [theme schemes](https://github.com/microsoft/fluentui/blob/f8113b88c86cccedd5f232640ef70c4e6282bca8/packages/theme/src/types/IScheme.ts#L15) feature. It is still marked as internal and subject to change. However, the [Theme-Designer](https://fluentuipr.z22.web.core.windows.net/heads/master/theming-designer/index.html) shows all the different schemas in the overview or is this something different?

Looking forward to some feedback 🤗

**Comment by micahgodbolt** [comment, marked as answer]
Schemes never did come out of internal testing. We are not using them in v9 either.
